packages/contracts-bedrock/src/L1/L1ERC721Bridge.sol

```diff
@@ -1,8 +1,9 @@
 // SPDX-License-Identifier: MIT
-pragma solidity 0.8.15;
+pragma solidity 0.8.25;
```

Comment:
This was too painful to manage the storage layout diff without using v5 oz initializable